.github/workflows/deploy.yml

```diff
@@ -0,0 +1,34 @@
+name: Deploy
+
+on: [ push, workflow_dispatch ]
+
+jobs:
+
+  affected:
+    uses: lunarmodules/.github/.github/workflows/list_affected_rockspecs.yml@main
```

Comment:
wyhyat does this do ?

Reply:
link for myself https://github.com/lunarmodules/.github

Reply:
Any time the rockspec files change and are pushed to the default branch, they are tested against several versions of Lua & LuaRocks to make sure they build properly. If they do, it publishes the rockspec to luarocks.

This particular line is a workflow fragment that figures out what, if any, rockspec files have been modified since the last push.